src/rtScriptDuk/rtScriptDuk.cpp

```diff
@@ -1072,7 +1073,7 @@ rtScriptDuk::rtScriptDuk():mRefCount(0)
   init();
 }
 
-rtScriptDuk::rtScriptDuk(bool initialize):mRefCount(0)
+rtScriptDuk::rtScriptDuk(bool initialize):mRefCount(0), node_is_initialized(false)
 #ifndef RUNINMAIN
```

Comment:
We should probably find another spot to initialize node_is_initialized to false.  Somewhere outside of the duktape sources.